werkit/compute/graph/_built_in_type.py

```diff
@@ -0,0 +1,25 @@
+import numbers
+import typing as t
+from typing_extensions import TypeGuard
+
+
+BuiltInValueType = t.Union[t.Type[bool], t.Type[int], t.Type[float], t.Type[str]]
+
+
+def is_built_in_value_type(_type: t.Type) -> TypeGuard[BuiltInValueType]:
+    return _type in (bool, int, float, numbers.Number, str)
```

Comment:
Why do you need `int`, `float` *and* `numbers.Number` here? Seems like it should be `int`, `float`, *or* `numbers.Number`. Also, why is `numbers.Number` not a part of the Union type in `BuiltInValueType`?